bugbug/models/regressor.py

```diff
@@ -49,6 +49,7 @@ def __init__(self, lemmatization=False, interpretable=False):
 
         feature_extractors = [
             commit_features.source_code_file_size(),
+            commit_features.source_code_file_metrics(),
```

Comment:
Don't add this yet to the two models, I want to add it in a separate PR so that we can measure the improvement in accuracy.